docs/Configuration.md

```diff
@@ -101,3 +101,17 @@ It is possible to mix global and local scope excludes in a single `.ort.yml`.
 
 For the available exclude reasons for scopes, see
 [ScopeExcludeReason.kt](../model/src/main/kotlin/config/ScopeExcludeReason.kt).
+
+For package manager specific scope excludes, see:
```

Comment:
Please add a hint that these are examples that need to be adapted to the projects, e.g.:
"See below examples for typical scope excludes for the supported package managers. Note that you have to verify that these examples match the configuration of your project."